_playwright-tests/UI/helpers/helpers.ts

```diff
@@ -20,6 +20,7 @@ export const closePopupsIfExist = async (page: Page) => {
 };
 export const filterByNameOrUrl = async (page: Page, name: string) => {
   await page.getByPlaceholder(/^Filter by name.*$/).fill(name);
+  expect(await page.getByText(name).count()).toBeGreaterThan(1);
```

Comment:
Do we actually want this to be more than one? :thinking: Or do we want the filter to have just one result?

Reply:
When we filter something, there are 3 components that can appear on the screen.
The chip, the row in the table, and the text in the input. 
The test picks up the chip only half the time, if the filter text is short enough. 

I suppose a better solution would be to use a locator reduction on the table, and check within the rows themselves...

... Just because you are right doesn't mean I like it... 😝